app/src/main/java/com/eveningoutpost/dexdrip/Services/G5CollectionService.java

```diff
@@ -282,11 +277,9 @@ public void startScan() {
 
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
             setupLeScanCallback();
-
             mBluetoothAdapter.startLeScan(new UUID[]{ BluetoothServices.Advertisement }, mLeScanCallback);
         } else {
             Log.d(TAG, "startScan");
-
             mLEScanner.startScan(filters, settings, mScanCallback);
```

Comment:
We should personally abstract out the BLE stuff, so an improvement in one will improve them all.  If it's possible that is.
Cheers

---- tzachi-dar wrote ----

>In app/src/main/java/com/eveningoutpost/dexdrip/Services/G5CollectionService.java:
>
>> mBluetoothAdapter.startLeScan(new UUID[]{ BluetoothServices.Advertisement }, mLeScanCallback); > } else { > Log.d(TAG, "startScan"); > - > mLEScanner.startScan(filters, settings, mScanCallback); 
>
>I guess that you are right, we need to catch that exception.
>
>—
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub￼
>